docs/home/SECURITY.md

```diff
@@ -3,6 +3,12 @@
 You are strongly advised use encrypted over unencrypted communication between Epiphany components where possible. Please consider
 this during planning your Epiphany deployment and configuration.
 
+We strongly advise to change default passwords wherever Epiphany configuration let you to do this.
+
+We strongly advise to use antivirus/antimalware software wherever possible to prevent risks. Please consider this during planning
```

Comment:
```suggestion
We strongly advise to use antivirus/antimalware software wherever possible to prevent security risks. Please consider this during planning
```